### ISSUE TITLE
Make Ctrl+Backspace delete the entire word

### ISSUE THREAD

**Issue by RMPR** [hiamandeep/speedcoder-keyboard-layouts#20]
I'm not really sure that here's is the best place to post this, but I think it 
will make things consistent between "real-world" programming and typing on the 
Website.

**Comment by mikeslattery** [comment]
Also [ctrl-w](http://vimdoc.sourceforge.net/htmldoc/insert.html#i_CTRL-W) for those that use vim or a vim plugin.

**Comment by RMPR** [comment]
I wonder if this will work since it's the keyboard shortcut to close a
tab.

**Comment by mikeslattery** [comment]
> I wonder if this will work since it's the keyboard shortcut to close a tab.

Good point.  I tried [several](https://stackoverflow.com/questions/24764626/any-way-to-prevent-disable-ctrl-key-shortcuts-in-the-browser/24764878) [tests](https://stackoverflow.com/questions/21695682/is-it-possible-to-catch-ctrlw-shortcut-and-prevent-tab-closing) and I couldn't trap ctrl-w on Firefox.

Not a big deal for me, as I map ctrl-backspace in `~/.vimrc` and `~/.ideavimrc`

    if has("gui_running")
        inoremap <C-bs> <C-w>
    else
        inoremap <C-h> <C-w>
    endif
